includes/admin/settings/class-wc-settings-accounts.php

```diff
@@ -84,7 +84,7 @@ public function get_settings() {
 				array(
 					'desc'          => __( 'When creating an account, automatically generate an account password.', 'woocommerce' ),
 					'id'            => 'woocommerce_registration_generate_password',
-					'default'       => 'no',
+					'default'       => 'yes',
```

Comment:
Should this not stay no since we are not wanting to default to creating accounts? Feel like setting this to yes will make it easier for someone to accidentally get an account created?

Reply:
"When creating an account" - account creation is no by default so this just makes it easier to set up an account IF it is enabled intentionally.